generator/generator.yml

```diff
@@ -101,3 +101,41 @@ modules:
       - 1.3.6.1.4.1.1718.3.2.2  # infeedTable
       - 1.3.6.1.4.1.1718.3.2.3  # outletTable
 
+# A10 Ax Load Balancers
+#
+# The MIB files are available for download through the load balancer GUI:
```

Comment:
@SuperQ you need to enter a valid device serial number during registration process.
@brian-brazil usually who wants to monitor a specific hardware has that.